assets/css/style.css

```diff
@@ -456,3 +453,596 @@ footer a {
   text-decoration: none;
 }
+
+/* mobile version */
+@media screen and (max-width: 768px) {
+  .navbar .desktop-menu {
+    display: none;
+  }
+
+  #workbg {
+    background-image: url(../images/EllipseFT.svg), url(../images/EllipseST.svg), url(../images/EllipseTT.svg), url(../images/EllipseFTT.svg);
+    background-repeat: no-repeat, no-repeat, no-repeat, no-repeat;
+    background-position-y: 12%, 37%, 62%, 86%;
+    background-position-x: 0%, 100%, 0%, 100%;
+  }
+
+  .contact-sec .headtwo {
+    display: none;
+  }
+
+  .surport-text .social-items .m1 {
+    display: none;
+  }
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  * {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    scroll-behavior: smooth;
+  }
+
+  body {
+    width: 1440px;
+    height: auto;
+    margin: 0 auto;
+    background: #191765;
+    padding: 10px;
+  }
+
+  .navbar {
+    width: 1440px;
+    height: 92px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .menubar {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    justify-content: space-around;
+    align-items: baseline;
+  }
+
+  .desktop-menu p {
+    color: #fff;
+    font-weight: 600;
+  }
+
+  .desktop-menu img {
+    width: 24px;
+    height: auto;
+  }
+
+  .desktop-menu img:hover {
+    background: rgb(150, 145, 145);
+  }
+
+  .desktop-menu .main-menu {
+    display: flex;
+    width: 241px;
+    height: 40px;
+  }
+
+  .desktop-menu .main-menu .menu-items {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    margin: 0 8px;
+    list-style: none;
+  }
+
+  .list-items li {
+    font-family: 'Roboto', sans-serif;
+    width: 68.08px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #f5f5ff;
+    display: flex;
+    align-items: center;
+    margin: 0 0;
+  }
+
+  .desktop-menu .main-menu .menu-items li {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    margin: 0 10px;
+  }
+
+  footer a {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: left;
+    width: 400px;
+    color: #ebebff;
+    text-decoration: none;
+  }
+
+  .desktop-menu .main-menu .menu-items .menu-item a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .headline {
+    width: 1156px;
+    height: auto;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    padding: 114px 0;
+    background: url(../images/circle1header.svg), url(../images/circle2header.svg);
+    background-repeat: no-repeat, no-repeat;
+    background-size: auto, auto;
+    background-position-y: 25%, 50%;
+    background-position-x: 0%, 110%;
+    animation-name: zoomin;
+    animation-duration: 3s;
+  }
+
```

Comment:
- [ ] Kindly decrease the space between sections following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) . Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/160214369-81caf1cf-9a51-41dc-b0d9-08fe9778247a.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/160214396-73ecc6a6-93d3-4f0f-aeb2-df187aee1f79.png)

Reply:
Applied suggested change